server/ideas-server/ideas-server/Program.cs

```diff
@@ -71,24 +71,25 @@ private static void InitializeDatabase(IWebHost host)
 
                 if (environment.IsDevelopment())
                 {
+
                     try
                     {
-                        services.InitializeInitiativeDatabase();
+                        services.InitiativeServiceBusEmlatorDatabase();
```

Comment:
Why the switch to the service bus emulator here?

Reply:
I changed the order to initiative the ServiceBusEmulator db first and then the InitiativeDatabase after (was the other way around).  git is just showing the changes a little funny.

The reason for the change is that other integration services depend on the ServiceBus db but not necessarily the initiatives db, so I want to get it done as first.

Reply:
Cool